adoc/code/mykernel.cpp

```diff
@@ -4,16 +4,13 @@
 // Explicit kernel names can be optionally forward declared at namespace scope
 class MyKernel;
 
-myQueue.submit([&](handler& h) {
-  // Explicitly name kernel with previously forward declared type
-  h.single_task<MyKernel>([=] {
+myQueue.single_task<MyKernel>([=] {
```

Comment:
You have also removed the calls to `submit` and replaced that with the shortcut function `queue::single_task`.  I don't think we want to do that here.  This section focuses on kernel naming, not the shortcut functions.  In addition, the example code just below this uses the `submit` style, so it would be better to keep a consistent style across the two examples.

Reply:
Oh ok. I think the other is verbose for no reason, and by putting verbose code an example, we encourage people to do so.

I will vote to change them all. As you said, we only care about the `naming` of the kernel, not how they are launched (via implicit or explicit cgh)

Reply:
It seems to me that we mostly use the separate `submit` / `parallel_for` style throughout the specification unless we are specifically illustrating the shortcut functions.  For example our canonical example in ["Example SYCL application"](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#_example_sycl_application) uses this style.

Reply:
Yes because we use buffer /accessor , so no other choice. We should change that too.

But I agree it's some work for another PR. Will reverse back to the submit  + x

Reply:
Done [9d88e6f](https://github.com/KhronosGroup/SYCL-Docs/pull/701/commits/9d88e6f7e3217ff29764dee3d198dafc2a98ae45)

Reply:
Thanks!